integration-cli/docker_cli_exec_test.go

```diff
@@ -43,37 +46,31 @@ func (s *DockerSuite) TestExecInteractiveStdinClose(c *check.C) {
 	if err != nil {
 		c.Fatal(err)
 	}
-
 	contId := strings.TrimSpace(out)
 
-	returnchan := make(chan struct{})
+	cmd := exec.Command(dockerBinary, "exec", "-i", contId, "echo", "-n", "hello")
+	p, err := pty.Start(cmd)
```

Comment:
If we don't have a way to do this today (even not using kr/pty), then yeah, let's move to _unix.go